app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/reliefdetails/ReliefDetailsController.scala

```diff
@@ -246,14 +246,19 @@ class ReliefDetailsController @Inject() (
             else {
               val updatedAnswers =
                 if (draftReturn.triageAnswers.isPeriodOfAdmin)
-                  answers.fold(
-                    _.copy(privateResidentsRelief = Some(AmountInPence.fromPounds(p))),
-                    _.copy(privateResidentsRelief = AmountInPence.fromPounds(p))
-                  )
+                  if (fillingOutReturn.isFurtherReturn.contains(true))
+                    answers.unset(_.privateResidentsRelief).unset(_.lettingsRelief)
```

Comment:
this should be left as it was before - letting relief is defaulted to a specific value for period of admin so we don't want to unset it